packages/autorag/Dockerfile.workspace

```
@@ -56,6 +57,9 @@ WORKDIR /usr/src/workspace/${UI_SOURCE_CODE}
 RUN npm ci --omit=optional --ignore-scripts
 
 # Build the module
+# Build app-config (provides webpack federation plugins)
+RUN cd /usr/src/workspace && npm run build -w packages/app-config
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)